infrastructure/tooling/src/generate/generators/docker-compose.js

```diff
@@ -240,6 +240,7 @@ exports.tasks.push({
       },
       volumes: [
         './db:/app/db', // in case of new migrations
+        './generated:/app/generated', // db schema
```

Comment:
should this be a different PR?

Reply:
I don't know. I can extract it if you feel like it should be better to have a separate PR for this.

Reply:
It is fine. I just wasn't sure what it was doing, and wasn't sure if it was related.

Reply:
yeah, it was required to make tings work locally. Let's keep it then, thanks!